src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1446,7 +1446,8 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
             {
                 var options = CreateCallOptions(headers: null, cancellationToken);
                 var response = await client.GetWorkflowAlpha1Async(request, options);
-                return new GetWorkflowResponse(response.InstanceId, response.StartTime, response.Metadata);
+                var dateTimeValue = new DateTime(response.StartTime);
```

Comment:
Let's use the [this constructor overload](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.-ctor?view=net-6.0#system-datetime-ctor(system-int64-system-datetimekind)) and specify `DateTimeKind.Utc`. Developers will appreciate this.